c2corg_ui/static/partials/cards/outings.html

```diff
@@ -10,10 +10,13 @@
     <span class="list-item-lang" ng-if="cardCtrl.lang != locale.lang">({{::locale.lang}})</span>
     <br>
     <div class="quality" ng-if="::doc['quality']" ng-class="::doc['quality']"
-         uib-tooltip="{{'quality'| translate}} : {{::doc['quality'] | translate}}" tooltip-placement="left"></div>
+         uib-tooltip="{{'quality'| translate}} : {{::doc['quality'] | translate}}" tooltip-placement="left">
+    </div>
 
-    <span class="elevation-max" ng-if="::doc['elevation_max']" uib-tooltip="{{'elevation_max' | translate}}">
-      <span translate class="value-title glyphicon glyphicon-sort-by-attributes"></span>
+    <!--elevation-max is shown only in the home feed-->
+    <span class="elevation-max" ng-if="::doc['elevation_max'] && cardCtrl.currentLocation == '/'"
```

Comment:
The info is in this sheet: https://docs.google.com/spreadsheets/d/1W2kEAyGPc7cw9yKe3hyYIMTNgUDLPsFUD0ccTS8jI5Y/edit?pref=2&pli=1#gid=0   -> elevation max on homepage only (but you are probably right that in profile feed probably too).

So I could test for  "/" and "/profiles/". Or what other testing would you suggest? this would seem robust enough to me, i cannot think of other cases where it would bring an error.